src/wabt.post.js

```diff
@@ -86,28 +100,24 @@ function allocateCString(s) {
 /// Features
 function Features(obj) {
   this.addr = Module._wabt_new_features();
-  for (var i = 0; i < FEATURES.length; ++i) {
-    var feature = FEATURES[i];
-    this[feature] = obj[feature] | 0;
-  }
+  allFeatures().forEach((v, f) => this[f] = booleanOrDefault(obj[f], v));
 }
 Features.prototype = Object.create(Object.prototype);
 
 Features.prototype.destroy = function() {
   Module._wabt_destroy_features(this.addr);
 };
 
-FEATURES.forEach(function(feature) {
-  Object.defineProperty(Features.prototype, feature, {
-    enumerable: true,
-    get: function() {
-      return Module['_wabt_' + feature + '_enabled'](this.addr);
-    },
-    set: function(newValue) {
-      Module['_wabt_set_' + feature + '_enabled'](this.addr, newValue | 0);
-    }
-  });
-});
+Object.keys(FEATURES).forEach(
+    feature => Object.defineProperty(Features.prototype, feature, {
+      enumerable: true,
+      get: function() {
+        return Module['_wabt_' + feature + '_enabled'](this.addr);
+      },
+      set: function(newValue) {
+        Module['_wabt_set_' + feature + '_enabled'](this.addr, newValue | 0);
+      }
+    }));
```

Comment:
Can you avoid re-indenting this entire block?  It looks like the just the first line is changing?